.eslintrc

```diff
@@ -0,0 +1 @@
+json { "extends": "mcr-codes", "env": { "browser": true } }
```

Comment:
If ESLint is not a dependency, there shouldnt be a config file for it included in the project